runtime/android/core_internal/src/org/xwalk/core/internal/XWalkLaunchScreenManager.java

```diff
@@ -572,8 +577,10 @@ private void hideLaunchScreenWhenReady() {
     }
 
     private void performHideLaunchScreen() {
-        mLaunchScreenDialog.dismiss();
-        mLaunchScreenDialog = null;
+        if (mLaunchScreenDialog != null) {
+            mLaunchScreenDialog.dismiss();
+            mLaunchScreenDialog = null;
+        }
```

Comment:
This change seems non related with your fix. If so, please seperate PR. And same for the style change, I'm glad to accept the style changes